harness/determined/util.py

```diff
@@ -259,11 +259,16 @@ def write_user_code(path: pathlib.Path, on_cluster: bool) -> None:
     # since it is rather common that users mount large, non-model files into their working directory
     # (like data or their entire HOME directory), when we are training on-cluster we use a
     # specially-prepared clean copy of the model rather than the working directory.
+    #
+    # When running locally, we filter out the files using detignore.
     if on_cluster:
         model_dir = constants.MANAGED_TRAINING_MODEL_COPY
+        ignore_func = shutil.ignore_patterns("__pycache__")
```

Comment:
Delete this line.

When we are on-cluster, we get the model definition from a special clean copy of the model definition, which can't have `__pycache__` because `__pycache__` is part of `DEFAULT_DETIGNORE`.

This line of code traces back to f14f769f91c where the src arg to copytree() was `os.getcwd()`.  I don't think it's relevant anymore, and I think it's safe to delete.